main.py

```diff
@@ -57,7 +57,7 @@ def loadTimeline(found):
     t = loadTimeline(found)
     cleanUpTimeline(t)
     saveTimeline(t)
-    Panel(window, height, width, t).create_panel()
+    Panel(window, height, width).create_panel()
```

Comment:
You know that the whole point of main is to check if a timeline object exist already:

- if it does it pulls it from pickle, and clean it up (if you havn't touched this app for more than a week)
- if it does not it creates a new timeline object
- either way it will save it to a pickle data file
-

Reply:
when i ran the code, this line was causing errors, because it was passing too many attributes to Panel